addons/rivet/devtools/rivet_cli.gd

```diff
@@ -3,53 +3,73 @@ extends RefCounted
-	printerr("Can't find path to Rivet CLI (rivet exec)")
-	
-	var rivet_cli_path = find_executable("rivet-cli")
-	if not rivet_cli_path.is_empty():
-		return rivet_cli_path
-	printerr("Can't find path to Rivet CLI (rivet-cli exec)")
-#endregion
+	if not editor_rivet_path or editor_rivet_path.is_empty():
+		return FAILED
+	var result: _RivetCliOutput = await run_command(["sidekick", "get-cli-version"])
+	if result.exit_code != 0 or !("Ok" in result.output):
+		return FAILED
+	var cli_version = result.output["Ok"].version
+	if cli_version != REQUIRED_RIVET_CLI_VERSION:
+		return FAILED
+	return OK
+
+func run_command(args: PackedStringArray) -> _RivetCliOutput:
+	var thread: _RivetThread = _RivetThread.new(_run.bind(args))
+	return await thread.wait_to_finish()
+
+func get_bin_dir() -> String:
+	var home_path: String = OS.get_environment("HOME")
+	return home_path.path_join(".rivet").path_join(REQUIRED_RIVET_CLI_VERSION).path_join("bin")
+
+func get_cli_path() -> String:
+	var cli_path = _RivetEditorSettings.get_setting(_RivetEditorSettings.RIVET_CLI_PATH_SETTING)
+	if cli_path and !cli_path.is_empty():
+		return cli_path
+	return get_bin_dir().path_join("rivet.exe" if OS.get_name() == "Windows" else "rivet")
+ 
+func install() -> _RivetCliOutput:
+	var thread: _RivetThread = _RivetThread.new(_install)
+	var result =  await thread.wait_to_finish()
+	if result.exit_code == 0:
+		_RivetEditorSettings.set_setting_value(_RivetEditorSettings.RIVET_CLI_PATH_SETTING, get_bin_dir())
+	return result
+
+
+## region Internal functions
 
 ## Runs Rivet CLI with given arguments.
-func run(args: PackedStringArray) -> _RivetCliOutput:
+func _run(args: PackedStringArray) -> _RivetCliOutput:
 	var output = []
-	var code = OS.execute(find_rivet(), args, output, true)
-	print("Running Rivet CLI: ", "rivet %s" % " ".join(args))
+	var code: int = OS.execute(get_cli_path(), args, output, true)
+	print("Running Rivet CLI: ", "%s %s" % [get_cli_path(), " ".join(args)])
 
 	return _RivetCliOutput.new(code, output)
 
-## Links your game with Rivet Cloud, using `rivet link` command.
-func run_command(args: PackedStringArray) -> _RivetCliOutput:
-	var thread := _RivetThread.new(run.bind(args))
-	
-	return await thread.wait_to_finish()
+func _install() -> _RivetCliOutput:
+	var output = []
+	var code: int
+	var bin_dir: String = get_bin_dir()
+
+	OS.set_environment("RIVET_CLI_VERSION", REQUIRED_RIVET_CLI_VERSION)
+	OS.set_environment("BIN_DIR", bin_dir)
+
+	# Double quotes issue: https://github.com/godotengine/godot/issues/37291#issuecomment-603821838
+	if OS.get_name() == "Windows":
+		var args = ["-Commandi",  "\"'iwr https://raw.githubusercontent.com/rivet-gg/cli/$env:RIVET_CLI_VERSION/install/windows.ps1 -useb | iex'\""]
+		code = OS.execute("powershell.exe", args, output, true)
+	else:
+		var args = ["-c", "\"'curl -fsSL https://raw.githubusercontent.com/rivet-gg/cli/${RIVET_CLI_VERSION}/install/unix.sh | sh\""]
+		OS.execute("/bin/bash", args, output, true)
```

Comment:
Can we change this to `/bin/sh`? I think it should be more ubiquitous, and the command that this shell will run is already using `sh`.